### PR TITLE
[new] Added validations to edit form 

### PR DESCRIPTION
**Pull request by apolevoi**
… all fields for the tutor and tutee edit forms. Also added a variable called tutor_items that is the label for checkboxes in the tutor model.

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -14,4 +14,27 @@ def after_sign_in_path_for(resource)
     end
   end
   
+  def validate_params(model, model_sym, specific_params)
```
Comment (junomorey) [inline app/controllers/application_controller.rb]: Not sure this is necessary - Rails already has built in validations you can just call in the model (e.g. "validates :first_name,  :last_name, etc..., presence :true)

Comment (apolevoi) [reply app/controllers/application_controller.rb]: I tried doing validates: etc. with all of the params but it threw a NoMethod error and I looked it up and saw that's a common problem with rails

Comment (apolevoi) [reply app/controllers/application_controller.rb]: Specifically, when I add this line to the top of tutees_controller:
 validates :first_name, :last_name, :sid, :grade, :email, :phone_number, :semesters_at_cal, :major, :requested_class, presence: true

I get:
undefined method `validates' for TuteesController:Class
on the local app when it's running at /tutees/1/edit

I followed this article specifically. https://apidock.com/rails/ActiveModel/Validations/ClassMethods/validates
Let me know if you can figure this out! If not, what I have works

**Comment by apolevoi** [comment]
@junomorey This refactoring has to do with @DhruvRelwani 's code in the controllers. What do you think, is it worth refactoring or do we push anyways?

File: app/controllers/tutees_controller.rb [hunk 2]
```diff
@@ -32,20 +23,26 @@ def edit
...
+                end
             end
-        end
-        flash[:notice] = "Form for #{@tutee.first_name + ' ' + @tutee.last_name} was succesfully created"
-        redirect_to tutor_match_path(@tutee)
+            flash[:notice] = "Form for #{@tutee.first_name + ' ' + @tutee.last_name} was succesfully created"
+            redirect_to tutor_match_path(@tutee)
+        rescue ActiveRecord::RecordInvalid => invalid   
+            flash[:error] = invalid.record.errors
+            redirect_to_correct_edit_form(@tutee, :tutee)
```
Comment (junomorey) [inline app/controllers/tutees_controller.rb]: I think you can just do 'redirect_to edit_tutee_path(@tutee)' here without the custom method

**Pull request merged**